package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "aframe-teleport-controls",
-  "version": "0.3.2",
+  "version": "0.3.3",
   "description": "A Teleport Controls component for A-Frame.",
   "main": "index.js",
   "unpkg": "dist/aframe-teleport-controls.min.js",
   "scripts": {
-    "dev": "budo index.js:dist/aframe-teleport-controls.min.js --port 7000 --live --open",
+    "dev": "budo index.js:dist/aframe-teleport-controls.min.js --port 7000 --live --open --ssl --cors --cert cert.crt --key key.key",
```

Comment:
it will probably make sense to switch to webpack with the `--https` parameter so it doesn't need a certificate, as currently this will break on people as they won't have the installed certificate locally